elasticapm/base.py

```diff
@@ -559,6 +565,13 @@ def _get_stack_info_for_trace(
             locals_processor_func=locals_processor_func,
         )
 
+    def _excepthook(self):
```

Comment:
Should this function also be calling `sys.__excepthook__` if it wasn't overridden? We probably want to maintain the default behavior of the exception system, after capturing the data for ourselves.